boards/xtensa/up_squared_adsp/tools/mbterm.py

```diff
@@ -14,8 +14,8 @@
 MBOX = 0x91281000
 LENGTH = 0x1000
 
-with open("/dev/mem", "r+b") as f:
-    mem_map = mmap(f.fileno(), LENGTH, offset=MBOX)
+with open("/dev/mem", "rb") as f:
+    mem_map = mmap(f.fileno(), LENGTH, access=ACCESS_COPY, offset=MBOX)
```

Comment:
I had to look this up (because **of course** Python doesn't expose the system's mmap syscall with the same API). 

"ACCESS_COPY" specifies a copy-on-write mapping.  That makes basically zero sense to me for a shared region.  Why on earth would this python script want its own copy of the memory that the other side was guaranteed not to see?  That's not a "permission", and I don't understand why you'd want it here at all.  What's going on?

Reply:
I have probably screwed something with this, should be `ACCESS_READ`, will fix.

Reply:
@andyross I cherry-picked this patch to #27804, addressed most of the comments except for this one, it needs some modification since from_buffer does not work with ACCESS_READ. I'll try to find some solution for this.